bin/common.sh

```diff
@@ -158,3 +158,48 @@ export_cachelib_lib_path() {
     CACHELIB_DIR=$STARROCKS_HOME/lib/cachelib
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CACHELIB_DIR/lib64
 }
+
+update_submodules()
```

Comment:
this is not necessary.

Reply:
yes, no shell script use it. So remove it ?

Reply:
of course you should. This is introduced by your backport